api/extensions.json

```diff
@@ -311,7 +311,7 @@
       },
       "extensionName": "minikube",
       "displayName": "minikube",
-      "shortDescription": "Integration for minikube: a tool that makes it easy to run Kubernetes locally",
+      "shortDescription": "Run Kubernetes locally",
```

Comment:
A little too generic? This could apply to OpenShift Local or Kind too. I know it's obvious beside the display name but feels like it could use something like 'Run Kubernetes locally with minikube' or something that's unique about minikube.

Reply:
At first I used with minikube but then in the catalog you see the name 'Minikube' and the description is to run Kubernetes so I would say it's the combination of the two. It feels redundant to say it's with minikube as it's the name of the extension

The GitHub page of minikube says `Run Kubernetes locally`